src/App.js

```diff
@@ -1,26 +1,33 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter , Route, NavLink } from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn
-        </a>
-      </header>
+
+
+
+const App = () => (
+  <BrowserRouter>
+    <div >
+      <a
+        href='https://webheroschool.ru/'
+        target='_blank'
+        rel='noopener noreferrer'
+
+      >
+        <div ></div>
+      </a>
+      <div >
+            <NavLink to='/todo' >
+          Дела
+        </NavLink>
+        <NavLink to='/' >
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8.](https://app.codacy.com/gh/WebHeroSchool/test_react/pullRequest?prid=5584351)